play-ad/src/main/scala/sbt/ad/SbtPlayAdPlugin.scala

```diff
@@ -2,32 +2,38 @@ package sbt.ad
 
 import sbt._
 import Keys._
-import play.{Play, PlayInternalKeys}
-
-object SbtAdPlayPlugin extends AutoPlugin with PlayInternalKeys {
+import play.Play
+import sbt.plugins.BackgroundRunPlugin
+import sbt.BackgroundJobServiceKeys
+import play.sbt.forkrun.{ PlayForkRun, PlayForkOptions }
+import play.forkrun.protocol.ForkConfig
+import PlayForkRun.autoImport._
+
+object SbtAdPlayPlugin extends AutoPlugin {
   import SbtAdPlugin.autoImport._
 
   object autoImport {
     val AppDynamicsPlay = config("appdynamicsplay").extend(AppDynamics)
-    val adPlayRunner = taskKey[BackgroundJobHandle]("Run play dev-mode runner with AppDynamics agent")
```

Comment:
`appdynamicsplay:backgroundRun` should start the new fork runner: https://github.com/typesafehub/sbt-ad/pull/2/files#diff-4b85a9d0d9627f9ab87b58a4eaff8286R29